### PR TITLE
Feature/generate keys

### PR DESCRIPTION
**Pull request by MatHenriquez**
# Summary
- Install the ***Stellar SDK***.
- Create a new folder containing files with ***helper functions***.
- Create the ***generateKeys.ts file*** in the helpers folder with a function to generate new keys.
- Change the Index componente to be a ***client component***.
- Import and use the ***generateKeys function*** in the Index component as a ***onClick event***.
- Import and use the ***useState React hook*** in the Index component to save the created keys.
- Create the ***handleCreateClick function*** to handle the ***click event*** on the 'Create new keys...' link.

# Details
- Add ***functionality*** to the ***Create new keys and Sign Up link*** to generate new keys using the ***stellar SDK***.
- Create a ***helper folder*** to store files with auxiliary functions and modularize the code.

# Evidence
## A new key pair generated.
generated_key_pair

### CODE REVIEW

File: app/helpers/generateKeys.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { Keypair } from "stellar-sdk";
+
+export function generateKeys() {
+  const keyPair: Keypair = Keypair.random();
+  return keyPair;
+}
```
Comment (Genebson) [inline app/helpers/generateKeys.ts]:
```suggestion
export function generateKeys() {
  return Keypair.random();
}
```

File: app/page.tsx [hunk 2]
```diff
@@ -1,6 +1,29 @@
...
+
 import { FC } from "react";
+import { generateKeys } from "./helpers/generateKeys";
+import { Keypair } from "stellar-sdk";
+import { useState } from "react";
+
+interface KeyPair {
+  publicKey: string;
+  secretKey: string;
+}
```
Comment (Genebson) [inline app/page.tsx]:
```suggestion
interface IKeyPair {
  publicKey: string;
  secretKey: string;
}
```

File: app/page.tsx [hunk 3]
```diff
@@ -1,6 +1,29 @@
...
+
+  function handleCreateClick(): void {
+    const newKeys: Keypair = generateKeys();
+
+    const generatedKeys: KeyPair = {} as KeyPair;
+    generatedKeys.publicKey = newKeys.publicKey();
+    generatedKeys.secretKey = newKeys.secret();
+
+    setKeys(generatedKeys);
+    console.log(generatedKeys);
```
Comment (Genebson) [inline app/page.tsx]: Delete this, please.

**Review by Genebson — APPROVED** [review]

File: app/page.tsx (hunk 3 above)
Comment (MatHenriquez) [reply app/page.tsx]: console.log deleted

File: app/page.tsx (hunk 2 above)
Comment (MatHenriquez) [reply app/page.tsx]: I have changed the interface name

File: app/helpers/generateKeys.ts (hunk 1 above)
Comment (MatHenriquez) [reply app/helpers/generateKeys.ts]: I have refactored this line

**Pull request merged**